bevy_lint/README.md

```diff
@@ -155,8 +155,9 @@ Once `bevy` is registered, you can toggle lints throughout your code, as long as
 // Enable pedantic lints, which are off by default.
 #![cfg_attr(bevy_lint, warn(bevy::pedantic))]
 
-// Deny methods of `World` in this system that can panic when a non-panicking alternative exists.
-#[cfg_attr(bevy_lint, deny(bevy::panicking_world_methods))]
+// Deny panicking `World` and `Query` methods in this system when a non-panicking alternatives
+// exist.
```

Comment:
Oooo that's a good point, I forgot about `QueryState`! 😅

Since we will likely add support for other panicking methods in the future, let's make it more general.

```suggestion
// Deny panicking Bevy methods in this system when a non-panicking alternatives exist.
```